packages/blocks/src/api/node.js

```diff
@@ -64,9 +58,9 @@ export function getNamedNodeMapAsObject( nodeMap ) {
  *
  * @throws {TypeError} If non-element/text node is passed.
  *
- * @param {Node} domNode DOM node to convert.
+ * @param {Element} domNode DOM node to convert.
```

Comment:
`Node` I believe is the correct type as typed in the JS. we allow `TEXT_NODE` and `ELEMENT_NODE` values for `nodeType`, of which the text node is not an element.

one can argue that in many of these cases we should restrict calling the function on anything else, but if we do that and relax the JS types then we'll end up hitting runtime crashes because of JS code that calls these without the benefit of type-checking.

Reply:
I am changing them back to `Node`, though it will need more consideration when/if type checking is enabled in this package.

Reply:
Also in `hpq` matchers are selectors on `string|Element`.